fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneRecordCursor.java

```diff
@@ -135,29 +136,29 @@
                         limitRemaining--;
                     }
                     List<Object> setPrimaryKey = Tuple.fromBytes(pk.bytes).getItems();
-                    List<Object> fields = Lists.newArrayList(fieldNames);
+                    List<Object> fieldValues = Lists.newArrayList(fields);
```

Comment:
I think it should be possible to do away with these dummy placeholders in the `IndexEntry` by adjusting the `AvailableFields`, and perhaps some other places. But a `TODO` for now is probably okay.